libra-node/src/main_node.rs

```diff
@@ -151,8 +149,14 @@ pub fn setup_network(
             config.enable_noise,
             "Permissioned network end-points must use authentication"
         );
-        let seed_peers = config.seed_peers.seed_peers.clone();
+        // Sanity check seed peer addresses.
+        config
+            .seed_peers
```

Comment:
better in the config space so we detect this earlier and in case it is used else where.